setup.py

```diff
@@ -242,9 +252,16 @@ def run(self):
                     PROTOC,
                     '--proto_path', SRC_DIR,
                     '--python_out', SRC_DIR,
+                    ] + protoc_mypy_args + [
                     '--cpp_out', SRC_DIR,
                     proto
                 ])
+                if gen_mypy:
+                    # Workaround for https://github.com/dropbox/mypy-protobuf/issues/8
+                    for file in os.listdir('onnx'):
```

Comment:
nit: don't use `file` as variable name, which is a builtin
use `SRC_DIR` instead of "onnx" (former is an absolute path)